host/class/uvc/usb_host_uvc_2/include/usb/uvc_host.h

```diff
@@ -0,0 +1,241 @@
+ * - This function should be called before calling any other UVC driver functions
+ *
+ * @param[in] driver_config Driver configuration structure. If set to NULL, a default configuration will be used.
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_install(const uvc_host_driver_config_t *driver_config);
+
+/**
+ * @brief Uninstall UVC driver
+ *
+ * - User must ensure that all UVC devices must be closed via uvc_host_stream_close() before calling this function
+ *
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_uninstall(void);
+
+/**
+ * @brief Open UVC compliant device
+ *
+ * @param[in]  stream_config  Configuration structure of the device
+ * @param[in]  timeout        Timeout in FreeRTOS ticks
+ * @param[out] stream_hdl_ret UVC stream handle
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_stream_open(const uvc_host_stream_config_t *stream_config, int timeout, uvc_host_stream_hdl_t *stream_hdl_ret);
+
+/**
+ * @brief Start UVC stream
+ *
+ * After this call, the user will be informed about new frames by frame callback
+ *
+ * @param[in] stream_hdl UVC handle obtained from uvc_host_stream_open()
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_stream_start(uvc_host_stream_hdl_t stream_hdl);
+
+/**
+ * @brief Stop UVC stream
+ *
+ * @param[in] stream_hdl UVC handle obtained from uvc_host_stream_open()
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_stream_stop(uvc_host_stream_hdl_t stream_hdl);
+
+/**
+ * @brief Pause UVC stream
+ *
+ * After this call, the USB transfers are not re-submitted and the user will stop getting new frame callbacks.
+ * This function does not issue any CTRL request. In only stops receiving new data from streaming endpoint.
+ *
+ * @param[in] stream_hdl UVC handle obtained from uvc_host_stream_open()
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_stream_pause(uvc_host_stream_hdl_t stream_hdl);
+
+/**
+ * @brief Start UVC stream
+ *
+ * After this call, the user will be informed about new frames by frame callback.
+ * This function does not issue any CTRL request. In only starts receiving new data from streaming endpoint.
+ *
+ * @param[in] stream_hdl UVC handle obtained from uvc_host_stream_open()
+ * @return esp_err_t
+ */
+esp_err_t uvc_host_stream_unpause(uvc_host_stream_hdl_t stream_hdl);
+
+/**
+ * @brief Close UVC device and release its resources
+ *
+ * @note All in-flight transfers will be prematurely canceled.
+ * @param[in] stream_hdl UVC handle obtained from uvc_host_stream_open()
+ * @return
+ *   - ESP_OK: Success - device closed
+ *   - ESP_ERR_INVALID_STATE: stream_hdl is NULL or the UVC driver is not installed
+ */
+esp_err_t uvc_host_stream_close(uvc_host_stream_hdl_t stream_hdl);
+
+/**
+ * @brief Return processed frame back to the driver
+ *
+ * In case your frame callback returns true, you MUST NOT call this function.
+ * In case your frame callback returns false, you MUS call this function after you are done with processing the frame.
```

Comment:
```suggestion
 * In case your frame callback returns false, you MUST call this function after you are done with processing the frame.
```

Even though, it hides in the text. Maybe it is better to start the sentence with it then. Something like: 

 ```suggestion
 * MUST be called, when frame callback returns false.
```